openreview/conference/builder.py

```diff
@@ -0,0 +1,349 @@
+        return options
+
+    def open_submissions(self, due_date = None, subject_areas = [], additional_fields = []):
+
+        ## Author console
+        authors_group = openreview.Group(id = self.id + '/Authors',
+            readers = ['everyone'],
+            signatories = [self.id],
+            signatures = [self.id],
+            writers = [self.id]
+        )
+        self.webfield_builder.set_author_page(self.id, authors_group, { 'title': 'Author console'})
+
+        ## Submission invitation
+        options = {
+            'name': 'Submission',
+            'due_date': due_date,
+            'reply': self.type.submission_reply(self.id, subject_areas, additional_fields)
+        }
+        return self.invitation_builder.set_submission_invitation(self.id, options)
+
+    def close_submissions(self, freeze_submissions = True):
+
+        # Get invitation
+        invitation = self.client.get_invitation(id = self.id + '/-/Submission')
+        if invitation.reply['writers'] != self.id:
+            invitation.reply['writers'] = self.id
+
+        now = round(time.time() * 1000)
+        if invitation.duedate > now:
+            invitation.duedate = now
+        self.client.post_invitation(invitation)
+
+        # if freeze submissions then remove writers
+        # use a process pool to run this
+        notes_iterator = tools.iterget_notes(self.client, invitation = invitation.id)
+        for note in notes_iterator:
+            if note.writers != [self.id]:
+                note.writers = [self.id]
+                self.client.post_note(note)
+
+        return invitation
+
+    def set_program_chairs(self, emails):
+        pcs_id = self.id + '/Program_Chairs'
+        return self.__create_group(pcs_id, self.id, emails)
+
+    def recruit_reviewers(self, emails, title = None, message = None, reviewers_name = 'Reviewers'):
+
+        pcs_id = self.id + '/Program_Chairs'
+        reviewers_id = self.id + '/' + reviewers_name
+        reviewers_declined_id = reviewers_id + '/Declined'
+        reviewers_invited_id = reviewers_id + '/Invited'
+        hash_seed = '1234'
+
+        self.__create_group(reviewers_id, pcs_id)
+        self.__create_group(reviewers_declined_id, pcs_id)
+        reviewers_invited_group = self.__create_group(reviewers_invited_id, pcs_id)
+
+        options = {
+            'reviewers_id': reviewers_id,
+            'reviewers_declined_id': reviewers_declined_id,
+            'hash_seed': hash_seed
+        }
+        invitation = self.invitation_builder.set_reviewer_recruiter_invitation(self.id, options)
+        invitation = self.webfield_builder.set_recruit_page(self.id, invitation, self.get_homepage_options())
+        recruit_message = '''Dear {name},
+
+        You have been nominated by the program chair committeee of ''' + self.id + ''' to serve as a reviewer.  As a respected researcher in the area, we hope you will accept and help us make the conference a success.
```

Comment:
in the past, we've used a variable called something like `short_phrase` for this kind of thing. So instead of using the ID `AKBC.ws/2019/Conference`, you could set `self.short_phrase = 'AKBC 2019'`, and that could be used here instead.

Reply:
I will add the short phrase field, thanks!